src/communication/Receiver.cpp

```diff
@@ -154,6 +156,41 @@ Receiver::Receiver(Uint16 port, std::string aetitle) {
     getConfig().addPresentationContext(dcmLongSCUStorageSOPClassUIDs[n], ts);
   }
   getConfig().addPresentationContext(UID_VerificationSOPClass, ts);
+  */
+  /*
+  OFList<OFString> ts1;
```

Comment:
Is there a reason why we decided to reduce the number of presentation contexts accepted? Was it not working with the old code? Because now we definitely have a very narrow set of presentation contexts that our receiver will accept

Reply:
Yeah, this decision comes from the lesson we've learned. In some cases, if both the sender and receiver have multiple syntaxes available, they might agree on just one of them. So when we pass in an image with another transfer syntax, it might be rejected. (see the story in #72). The current strategy is to add minimal presentation contexts to ensure the current pipeline and unit tests work on the current datasets, and develop #72 in the future.